doc/common/abbreviations.md

```diff
@@ -315,6 +327,47 @@
 <a name="1.3"></a>
 ## RA Abbreviations
 
+> The abbreviations in this section are in addition to the [Common Abbreviations](#1.1) and are specific to the Reference Architectures (RA1 and RA2).
+> 
+| Term | Description |
+| ------------- | --------------------- |
+| ARP | Address Resolution Protocol  |
+| BGP | Border Gateway Protocol |
+| BGPaaS | BGP as a Service |
+| CCP | Centralised Control Plane |
+| cgroups | Control Groups |
+| CM | Configuration Management |
+| CRD | Custom Resource Definition |
+| CRI-O | OCI compliant CRI |
```

Comment:
CRI-O does not mean OCI compliant CRI, but it is an OCI compliant container runtime implementation,

Reply:
@CsatariGergely Sorry -- Done (OCI compliant CRI implementation as per https://cri-o.io/. Thanks

Reply:
@CsatariGergely if you are OK with the change can you please resolve. Thanks